tests/functional/test_accounts.py

```diff
@@ -60,6 +60,24 @@ def test_transfer(sender, receiver):
     assert receiver.balance == expected
 
 
+def test_transfer_without_value(sender, receiver):
+    with pytest.raises(AssertionError) as err:
+        sender.transfer(receiver)
+    assert str(err.value) == "Transfer without value argument requires kwarg send_everything=True"
+
+
+def test_transfer_without_value_send_everything_false(sender, receiver):
+    with pytest.raises(AssertionError) as err:
+        sender.transfer(receiver, send_everything=False)
+    assert str(err.value) == "Transfer without value argument requires kwarg send_everything=True"
+
+
+def test_transfer_without_value_send_everything_true(sender, receiver):
+    initial_balance = receiver.balance
+    sender.transfer(receiver, send_everything=True)
+    assert receiver.balance > initial_balance, "Receiver has same balance after transfer"
```

Comment:
Can as assert on the actual expected value to make we are properly accounting for fees?

Reply:
this was something I was having trouble trying to figure out, since the fee seems different at times and I'm not sure how that's calculated. maybe i can use txn.total_cost?

Reply:
Total_transfer_value? Hmmmm

Reply:
total_transfer_value does not work, if fails with the error that the gas is higher than that number.

Reply:
max_fee also doesn't work for the same reason

Reply:
Ok! How about we assert that the sender has no more balance?

Reply:
This doesn't work because gas fees are overestimated (discussed in this morning's meeting) but I did assert that the sender has less than 1 finney now.